### ISSUE TITLE
infer-and-annotate.sh may require checker-quals.jar on classpath

### ISSUE THREAD

**Issue by jyoo980** [typetools/checker-framework#6239]
Running `infer-and-annotate.sh` on [`multi-version-control`](https://github.com/plume-lib/multi-version-control/tree/master) yields the following error(s): https://gist.github.com/jyoo980/ea9cd174057ec9aa0bdc122b4b4587cc

Specifically, the script complains about not being able to load the `@Nullable` class:

> Could not load class: org.checkerframework.checker.nullness.qual.Nullable

**Steps to Reproduce**

1. Clone  [`multi-version-control`](https://github.com/plume-lib/multi-version-control/tree/master) and build it, run: `./gradlew assemble`.
2. Execute `infer-and-annotate.sh` with the following command, adjust paths as necessary (e.g., replace `PATH_TO_MULTI_VERSION_CONTROL` the path on your own machine)
```sh
infer-and-annotate.sh "NullnessChecker" $CHECKERFRAMEWORK/checker/dist/checker-qual.jar:<PATH_TO_MULTI_VERSION_CONTROL>/build/libs/multi-version-control.jar:<PATH_TO_MULTI_VERSION_CONTROL>/build/libs/multi-version-control-all.jar $(find src/main/java/org/plumelib/multiversioncontrol/ -name "*.java")
```

The script should finish with no errors with loading Checker Framework annotations, since the classpath includes them (` $CHECKERFRAMEWORK/checker/dist/checker-qual.jar`).

Relevant portion of the Checker Framework manual: https://checkerframework.org/manual/#wpi-insert
